ETABS_oM/Requests/Enum/PierAndSpandrelResultType.cs

```diff
@@ -27,7 +27,8 @@ namespace BH.oM.Adapters.ETABS.Requests
     [Description("Defines the type of results that should be extracted for PierAndSpandrelForceRequest.")]
     public enum PierAndSpandrelResultType
     {
-        PierForce
+        PierForce,
+        SpandrelForce
```

Comment:
I don't think this `Enum` is needed - see below comments on `BarResultRequest`.

Reply:
Not 100% clear to me. Happy to discuss about it as soon as you're free. I understand this is not crucial for the code to work as expected. I might resolve this to allow for running BHoM Bot checks updating this asap in the next milestone.